src/gsi-ncdiag/gsi_ncdiag.py

```diff
@@ -782,6 +782,12 @@ def toIODAobs(self, OutDir, clobber=True, platforms=None):
                             else:
                                 tmp[tmp > 4e8] = self.FLOAT_FILL
                             outdata[gvname] = tmp
+                    # create a GSI effective QC variable
+                    gsiqcname = outvars[o], 'GsiEffectiveQC'
+                    errname = outvars[o], 'GsiFinalObsError'
```

Comment:
I believe it's, but certainly, the others can say about it better: is these criteria based on obs errors sufficient to encapsulate the conventional & ozone cases too (not only radiances)?